docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md

```diff
@@ -142,11 +144,14 @@ metadata:
   resourceVersion: ""
   selfLink: ""
 ```
+{% endcapture %}
+
+{% capture discussion %}
+## Advanced topics
 
-## Advanced Topics
 ### Finalizers
-CustomResources (objects created in the schema defined by CustomResourceDefintions)
-support finalizers.  If you add a `metadata.finalizers` stanza like
+
+Custom objects support finalizers. If you add a `metadata.finalizers` stanza like:
```

Comment:
Hmm...I see what you mean, and am wondering if a reordering would work to provide more context, along the lines of:

> Custom objects support *finalizers*, which behave similarly to pre-delete hooks. Finalizers can be used for cleanup of custom objects, just like with built-in objects.
>
> You can do so by adding a `metadata.finalizers` stanza to the YAML of your custom object:
> `...STUFF...`
>
> When the object is deleted, its `metadata.deletionTimestamp` will be set and *update* watch events will be sent to a controller. The controller performs any finalization logic, before removing the finalizer and deleting the object again. 

If the "hook" explanation is not 100% accurate or useful, you can also just say "Custom objects support *finalizers*, which can be used for cleanup of custom objects similar to that of built-in objects."